Providers/Makefile

```diff
@@ -100,6 +100,13 @@ PROVIDERS+=nxFileInventory
 PROVIDERS+=nxOMSAutomationWorker
 endif
 
+# list of providers
+ifeq ($(BUILD_OMS),BUILD_OMS)
+TEST_PROVIDERS:=nxNop
+else
+TEST_PROVIDERS:=nxNop
+endif
```

Comment:
I am removing this for now as we don't need to build test providers always. And there are some issue that we need to fix in python script for this provider so will enable it once that is done. 

---
In reply to: [107557794](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/235#discussion_r107557794) [](ancestors = 107557794)